front/src/components/LoginPanel/comp/LoginForm.js

```diff
@@ -1,4 +1,5 @@
-import React, { useState, useEffect } from "react";
+import React, { useState } from "react";
+import "./LoginForm.css";
 import Button from "../../UI/Button/Button";
 import { gql, useMutation } from "@apollo/client";
 
```

Comment:
Lines 18-22:
```
    const [loginUser, { data }] = useMutation(LOGIN_USER, {
        onError(err) {
            console.log(err);
        },
    });
```
Change this to something visible to user, even alert would be better.